frontend/web/src/app/services/digital-part/digital-part.service.ts

```diff
@@ -36,6 +35,12 @@ export class DigitalPartService {
           return data;
       });
   }
+  uploadStlFile(digitalPart: DigitalPart, stlFile: any): Observable<string> {
+    return this.client.post(this.endpoint + '/' + digitalPart.id + '/stl', stlFile)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a template literal instead of concatenating with a string literal. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS3512)